docs/sites.yml

```diff
@@ -4431,3 +4431,15 @@
     - Featured
   built_by: Algolia
   featured: true
+- title: GVD Renovations
+  url: https://www.gvdrenovations.com/
+  main_url: https://www.gvdrenovations.com/
+  description: >
+    GVD Renovations is a home improvement contractor with a well know reputation as professional, quality contractor in California.
+  categories:
+    - Home Improvement
```

Comment:
Could you do me a quick favor and ensure that these categories are more generally applicable?

In other words--if you check out https://www.gatsbyjs.org/showcase/, these categories should have appeared before if at all possible! We want to keep these as "general" as possible, so in this case I'd recommend something like:

```yaml
categories:
  - Business
  - Real Estate
```

and then any other that you think fit! Please let me know if you have any questions on this!

Reply:
I see. Let's go with 

- Construction
- Business

Reply:
Works for me!